client/post-editor/editor-sidebar/index.jsx

```diff
@@ -27,6 +28,11 @@ export default class EditorSidebar extends Component {
 		isPrivate: PropTypes.bool,
 	}
 
+	_viewFeedbackPane = () => {
```

Comment:
It is there because of an old habit of marking members intended to be private and because I saw it [used](https://github.com/Automattic/wp-calypso/blob/f97ae1a398aa1f5ad210877b7a58a68dc2ca384e/client/components/dialog/dialog-base.jsx#L42) in the Calypso codebase. But I'm glad you asked because I'm not sure it is useful when used with React. It's not common that we directly call methods on other components.

I plan to remove the underscore and avoid the practice in other components, unless someone knows a good reason we should continue.